adhocracy4/filters/filters.py

```diff
@@ -2,7 +2,22 @@
 from django.http import QueryDict
 
 
-class DefaultsFilterSet(django_filters.FilterSet):
+class PagedFilterSet(django_filters.FilterSet):
+    """Removes page parameters from the query when applying filters."""
+    page_kwarg = 'page'
+
+    def __init__(self, data, *args, **kwargs):
+        if self.page_kwarg in data:
+            try:
+                data.pop(self.page_kwarg)
+            except AttributeError:
+                # Create a mutable copy
+                data = data.copy()
+                data.pop(self.page_kwarg)
```

Comment:
@slomo what do you mean by "there is also QueryDict.mutable"
i did consider the mutable case as default because i assumed the `PagedFilterSet` will be called mainly as the parent class of `DefaultsFilterSet` which is explicitly using a mutable QueryDict
unfortunately the QueryDict is using the private `_mutable` attribute for its state and has no public method to get it's state. i did consider _underscore attributes as not suitable for use